circuits/cpp/src/aztec3/utils/types/circuit_types.hpp

```diff
@@ -9,43 +9,40 @@ using namespace proof_system::plonk;
 
 namespace aztec3::utils::types {
 
-template <typename Composer> struct CircuitTypes {
+template <typename Builder> struct CircuitTypes {
     // Basic uint types
-    using boolean = stdlib::bool_t<Composer>;
-    using uint8 = stdlib::uint8<Composer>;
-    using uint16 = stdlib::uint16<Composer>;
-    using uint32 = stdlib::uint32<Composer>;
-    using uint64 = stdlib::uint64<Composer>;
+    using boolean = stdlib::bool_t<Builder>;
+    using uint8 = stdlib::uint8<Builder>;
+    using uint16 = stdlib::uint16<Builder>;
+    using uint32 = stdlib::uint32<Builder>;
+    using uint64 = stdlib::uint64<Builder>;
 
     // Types related to the bn254 curve
-    using fr = stdlib::field_t<Composer>;
-    using safe_fr = stdlib::safe_uint_t<Composer>;
-    using address = stdlib::address_t<Composer>;
-    using fq = stdlib::bigfield<Composer, barretenberg::Bn254FqParams>;
+    using fr = stdlib::field_t<Builder>;
+    using safe_fr = stdlib::safe_uint_t<Builder>;
+    using address = stdlib::address_t<Builder>;
+    using fq = stdlib::bigfield<Builder, barretenberg::Bn254FqParams>;
 
-    using witness = stdlib::witness_t<Composer>;
+    using witness = stdlib::witness_t<Builder>;
 
     // typedef fq grumpkin_fr;
     // typedef fr grumpkin_fq;
-    using grumpkin_point = stdlib::point<Composer>;  // affine
-    using grumpkin_group = stdlib::group<Composer>;
+    using grumpkin_point = stdlib::point<Builder>;  // affine
+    using grumpkin_group = stdlib::group<Builder>;
 
-    using bn254 = stdlib::bn254<Composer>;
+    using bn254 = stdlib::bn254<Builder>;
     // typedef bn254::g1_ct bn254_point;
-    using bn254_point = stdlib::element<Composer, fq, fr, barretenberg::g1>;  // affine
+    using bn254_point = stdlib::element<Builder, fq, fr, barretenberg::g1>;  // affine
 
-    using bit_array = stdlib::bit_array<Composer>;
-    using byte_array = stdlib::byte_array<Composer>;
-    using packed_byte_array = stdlib::packed_byte_array<Composer>;
+    using bit_array = stdlib::bit_array<Builder>;
+    using byte_array = stdlib::byte_array<Builder>;
+    using packed_byte_array = stdlib::packed_byte_array<Builder>;
 
-    using schnorr_signature = stdlib::schnorr::signature_bits<Composer>;
-    using ecdsa_signature = stdlib::ecdsa::signature<Composer>;
+    using schnorr_signature = stdlib::schnorr::signature_bits<Builder>;
+    using ecdsa_signature = stdlib::ecdsa::signature<Builder>;
 
     using AggregationObject = stdlib::recursion::aggregation_state<bn254>;
-    using recursive_inner_verifier_settings =
-        std::conditional_t<std::same_as<Composer, plonk::TurboPlonkComposer>,
-                           stdlib::recursion::recursive_turbo_verifier_settings<bn254>,
-                           stdlib::recursion::recursive_ultra_verifier_settings<bn254>>;
+    using recursive_inner_verifier_settings = stdlib::recursion::recursive_ultra_verifier_settings<bn254>;
```

Comment:
Is his because we don't expect turbo to be used as inner circuit builder anymore? If so, makes sense.

Reply:
Well, Luke's previous PR fixed the inner circuit type to Ultra (indeed, `Aggregate::aggregate` now uses a fixed flavor), so it wouldn't even work to toggle the inner type. In general this recursion stuff needs a rework, so I thought I'd pare it down to make it simpler for when we do that work. In fact, I see now this alias is not in use anywhere, so I think I'll remove it.